src/components/CompareResults/TableHeader.tsx

```diff
@@ -134,7 +134,7 @@ function FilterableColumnHeader({
         </Box>
         <KeyboardArrowDownIcon />
       </Button>
-      <Menu {...bindMenu(popupState)}>
+      <Menu sx={{ zIndex: 2000 }} {...bindMenu(popupState)}>
```

Comment:
Hmm, is this the only place we need this because I noticed the tooltips aren't in the subtests table view either. It might help to add this to the Menu MUI overrides in the component file